packages/frontend/editor-ui/src/stores/users.store.ts

```diff
@@ -219,13 +206,22 @@ export const useUsersStore = defineStore(STORES.USERS, () => {
 		setCurrentUser(user);
 	};
 
+	const registerLoginHook = (hook: LoginHook) => {
+		loginHooks.value.push(hook);
+	};
+
+	const registerLogoutHook = (hook: LogoutHook) => {
+		logoutHooks.value.push(hook);
+	};
+
```

Comment:
Not familiar with Vue best practices - how do we ensure that folks will use these instead of pushing to `loginHooks` and `logoutHooks` directly?